components/settings/actions/ActionDialog.js

```diff
@@ -47,7 +54,7 @@ export default class ActionDialog extends React.Component {
                         GEPPETTO.trigger(GEPPETTO.Events.Hide_spinner);
                         this.props.onRequestClose();
                     }
-            });
+						});
```

Comment:
btw, shouldn't we remove any use of 'NeuroML'? There is at least an 'if' clause using it in this file